### ISSUE TITLE
Windows 11: The Group Policy settings for Bitlocker startup options are in conflict

### ISSUE THREAD

**Issue by WinterPhoenix** [simeononsecurity/Windows-Optimize-Harden-Debloat#85]
**Describe the bug**
Apparently if **Require TPM** under **Configure TPM startup** is set, it's in conflict, even though it worked fine in Windows 10:

![Screenshot 2024-01-26 234324](https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/assets/2799393/f38d2dee-bc05-4397-8283-ad31e7e7b384)

![Screenshot 2024-01-26 234407](https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/assets/2799393/eaab30d9-2f8c-4b78-91a5-b42f55848ea7)

**To Reproduce**
1. Install Windows 11 (Pro?)
    - Ideally without Bitlocker automatic encryption enabled, so it doesn't encrypt with inferior defaults and also break after the script it applied
3. Run sos-optimize-windows.ps1 v4.0.2 with bitlockerhardening enabled
4. Try to **Turn on Bitlocker** for your Operating System drive

**Expected behavior**
Bitlocker encryption works as normal.

**Solution**
Setting **Configure TPM startup** to **Allow TPM** after applying the optimize script rectifies the issue.

**Comment by simeononsecurity** [comment]
This is a valid issue. We'll need to review the reasons why we did it that way and see if something has changed since we implemented it. We believe the exact setting came from the STIG baselines we mirror and apply. But it wouldn't surprise us if those, if configured literally, would run into issues. Though we normally would catch this.
Expect a fix in a few days or a week at most.  We'll have to review our update and review process along with this.

Thanks for the submission @WinterPhoenix .
